### ISSUE TITLE
Wrong width

### ISSUE THREAD

**Issue by usb248** [syropian/vue-input-autowidth#16]
Hi,

On last version FF, since last update.

My input seems to appear with confortZone > 0 while ... :
`v-autowidth="{maxWidth: '680px', minWidth: '50px', comfortZone: 0}"`

![image](https://user-images.githubusercontent.com/7659927/64632153-d7f4b700-d3f8-11e9-9494-ac5db5a63d93.png)

Please fix 🤔

**Comment by syropian** [comment]
Could you reproduce the issue using something like CodeSandbox please?

**Comment by usb248** [comment]
I can but not immediately. You don't succeed reproduce the issue ?

**Comment by syropian** [comment]
@usb248 No, I can't. The demo page seems to be working correct in Firefox for me - https://syropian.github.io/vue-input-autowidth/

Do you have a large amount of right padding on the input?

**Comment by usb248** [comment]
No, any right padding otherwise this display should be okay...exactly.
This is perhaps the font .. ?

**Comment by usb248** [comment]
This is not working even on your demo page (windows 10, last version of FF):
![image](https://user-images.githubusercontent.com/7659927/64681051-4bd9a280-d47f-11e9-91cf-f633be39c626.png)

**Comment by syropian** [comment]
Ah interesting, it seems to _only_ affect Firefox on Windows 10. Firefox on Mac is fine,

**Comment by syropian** [comment]
Ok, I just pushed a fix and released 1.0.8. Please upgrade to that version, it should fix your issue. I can confirm the demo page looks correct on FF in Windows 10.
